### PR TITLE
Updated WebUI config options in docs to support direct linking to installs

### PR DESCRIPTION
**Pull request by laf**
Updated documentation into new format for quick access to peoples installs.

Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: doc/Extensions/Distributed-Poller.md [hunk 1]
```diff
@@ -41,23 +41,26 @@ These requirements are above the normal requirements for a full LibreNMS install
...
+!!! setting "poller/distributed"
+    ```bash
+    lnms config:set distributed_poller true
+    lnms config:set distributed_poller_group 0
+    ```
+
+If you want to customise the hostname for the poller then you will need
+to set this in `config.php`:
+
+`lnms config:set distributed_poller_name 'custom'`
```
Comment (murrant) [inline doc/Extensions/Distributed-Poller.md:62]: This one actually needs to be set in config.php otherwise it applies to all nodes.

Comment (Jellyfrog) [reply doc/Extensions/Distributed-Poller.md:62]: it should go in .env in that case?

Comment (murrant) [reply doc/Extensions/Distributed-Poller.md:62]: yes, and there is an .env for the dispatcher, but not the python wrapper.

Maybe it is time to deprecate using the python wrapper for distributed polling?

**Comment by murrant** [comment]
Looks good overall.  Two thoughts:
1. Did you test running every lnms config:set command?  If not I can.
2. I noticed we lost some variable documentation in comments.  I think it might be an good idea to implement `lnms config:help` to spit out the translatable help text from the webui.

File: doc/Extensions/Distributed-Poller.md (hunk 1 above)
Comment (laf) [reply doc/Extensions/Distributed-Poller.md]: Well spotted, I've reverted that change. Didn't change it elsewhere, must have just got over zealous with this one!

**Review by murrant — APPROVED** [review]

**Review by murrant — APPROVED** [review]

**Pull request merged**